src/compiler/scala/tools/nsc/symtab/classfile/ClassfileParser.scala

```diff
@@ -376,7 +376,9 @@ abstract class ClassfileParser {
     //   - remove eager warning
     val msg = s"Class $name not found - continuing with a stub."
     if ((!settings.isScaladoc) && (settings.verbose || settings.developer)) warning(msg)
-    NoSymbol.newStubSymbol(name.toTypeName, msg)
+    val stub = NoSymbol.newStubSymbol(name.toTypeName, msg)
+    stub.pos = NoPosition
```

Comment:
We can use `lastTreeToTyper.pos` here, but I haven't found any example in which it is necessary.